### DISCUSSION TITLE
Large Windows(tm) executables

### DISCUSSION THREAD

**Discussion by ghost** [jamulussoftware/jamulus#2101] [Technical Support & Help] [answered]
Why are jamulus_3.8.1_win_jack.exe and jamulus_3.8.1_win.exe 75 megabytes in size? Are they not stripped? This could be a performance burden on Windows. 75MB is ten times larger than reasonable

**Comment by pljones** [comment]
Those are the installer names.  Have you checked the installed exes?

**Comment by henkdegroot** [comment, marked as answer]
One of the main reasons is that everything is in there twice. One for deployment on a 32-bit only setup and one for deployment to 32/64-bit. And because of this the vc-runtime lib (approx. 14 Mb in size) is included twice as well.
What I do find odd is that my local build installer is 67 Mb and the official build is 75 Mb. Think this is most likely due to difference in windows redist versions, etc.
The jamulus.exe itself is just approx. 2 Mb.

**Reply by ghost** [reply]
Thanks for the explanation.
